### PR TITLE
fix readme cmds for clip-roberta

### PR DESCRIPTION
**Pull request by hsubramony**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/contrastive-image-text/README.md [hunk 1]
```diff
@@ -128,33 +128,34 @@ PT_HPU_LAZY_MODE=0 python run_clip.py \
...
-    --image_column image_path \
-    --caption_column caption \
-    --remove_unused_columns=False \
-    --do_train  --do_eval \
-    --per_device_train_batch_size="512" \
+PT_HPU_LAZY_MODE=0 PT_ENABLE_INT64_SUPPORT=1 \
+python3 ../gaudi_spawn.py --world_size 8 --use_mpi run_clip.py \
+    --output_dir=/tmp/clip_roberta \
+    --model_name_or_path=./clip-roberta \
+    --data_dir=./data \
```
Comment (libinta) [inline examples/contrastive-image-text/README.md:135]: should we change this back?

File: examples/contrastive-image-text/README.md [hunk 2]
```diff
@@ -128,33 +128,34 @@ PT_HPU_LAZY_MODE=0 python run_clip.py \
...
+    --dataset_name="ydshieh/coco_dataset_script" \
+    --dataset_config_name="2017" \
+    --image_column="image_path" \
+    --caption_column="caption" \
```
Comment (libinta) [inline examples/contrastive-image-text/README.md:139]: no need to change 136-139

File: examples/contrastive-image-text/README.md [hunk 3]
```diff
@@ -128,33 +128,34 @@ PT_HPU_LAZY_MODE=0 python run_clip.py \
...
+    --remove_unused_columns="False" \
```
Comment (libinta) [inline examples/contrastive-image-text/README.md:140]: no need to change 140

**Comment by srajabos** [comment]
Verified the command for deep speed, LGTM.

File: examples/contrastive-image-text/README.md [hunk 4]
```diff
@@ -128,33 +128,34 @@ PT_HPU_LAZY_MODE=0 python run_clip.py \
...
-    --throughput_warmup_steps 3 \
-    --dataloader_num_workers 16 \
-    --mediapipe_dataloader \
-    --bf16 \
-    --sdp_on_bf16 \
-    --distribution_strategy fast_ddp \
-    --trust_remote_code \
+    --use_lazy_mode=False \
+    --gaudi_config_name="Habana/clip" \
+    --throughput_warmup_steps=30 \
```
Comment (shepark) [inline examples/contrastive-image-text/README.md:150]: @hsubramony  is this 30 correct? just double check..

**Review by libinta — APPROVED** [review]

File: examples/contrastive-image-text/README.md (hunk 4 above)
Comment (regisss) [reply examples/contrastive-image-text/README.md:150]: I agree with @shepark that it looks weird

**Pull request merged**